csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -1613,11 +1613,13 @@ The value MUST be one of the following:
     interim
 ```
 
-The value `draft` indicates, that this is a pre-release, intended for issuing party’s internal use only, or possibly used externally when the party is seeking feedback or indicating its intentions regarding a specific issue.
+The value `draft` indicates, that this is a pre-release, intended for issuing party's internal use only, or possibly used externally when the party is seeking feedback or indicating its intentions regarding a specific issue.
 
-The value `final` indicates, that the issuing party asserts the content is unlikely to change. “Final” status is an indication only, and does not preclude updates.
+The value `final` indicates, that the issuing party asserts the content is unlikely to change. “Final” status is an indication only, and does not preclude updates. This SHOULD be used if the issuing party no, slow or few changes.
```

Comment:
There appears to be a typo that blocks me from understanding the intent of the sentence:
> [...]if the issuing party no, slow or few changes.

Reply:
I hope the inserted word makes it clear. :-)